### PR TITLE
CLI: `hanami version` rewrite

### PR DESCRIPTION
**Pull request by jodosha**
Implement `hanami version` with the new `hanami-cli` gem.

### CODE REVIEW

File: bin/hanami [hunk 1]
```diff
@@ -1,5 +1,8 @@
...
-
 require 'bundler'
-require 'hanami/cli'
-Hanami::Cli.start
+require 'hanami/command_line'
+
+result = Hanami::CommandLine.run
+
+exit(0) if result
+Hanami::OldCommandLine.start
```
Comment (davydovanton) [inline bin/hanami]: awesome 👏

File: lib/hanami/command_line.rb [hunk 2]
```diff
@@ -1,26 +1,28 @@
...
+  module CommandLine
+    include Hanami::Cli
+
+    class Version
+      def call
+        puts "v#{Hanami::VERSION}"
+      end
+    end
+
+    register "version", Version
```
Comment (davydovanton) [inline lib/hanami/command_line.rb]: What will happen if I call `hanami v` or `hanami --version`?

Comment (davydovanton) [inline lib/hanami/command_line.rb]: and I understand right that command description will we create in future?

File: lib/hanami/command_line.rb [hunk 3]
```diff
@@ -1,26 +1,28 @@
...
 require 'hanami/commands/new/app'
 require 'hanami/commands/new/container'
 
 module Hanami
+  module CommandLine
+    include Hanami::Cli
+
+    class Version
+      def call
+        puts "v#{Hanami::VERSION}"
```
Comment (AlfonsoUceda) [inline lib/hanami/command_line.rb]: I think we need `require 'hanami/version'` isn't it?

File: lib/hanami/command_line.rb (hunk 2 above)
Comment (AlfonsoUceda) [inline lib/hanami/command_line.rb]: @davydovanton for now hanami-cli is very naive, this is a PR for testing the new Cli, I think we shouldn't merge yet.

File: lib/hanami/command_line.rb (hunk 3 above)
Comment (jodosha) [inline lib/hanami/command_line.rb]: It's automatically required by `hanami.gemspec`. I think it's fine.

**Comment by jodosha** [comment]
@AlfonsoUceda If you're talking about `--version` and `-v` aliases, I'm fine to wait for them. If so, just push new commits here.

**Comment by AlfonsoUceda** [comment]
@jodosha I've implemented aliases for -v and --version

**Comment by jodosha** [comment]
@AlfonsoUceda @oana-sipos As long you will merge hanami/cli#2, please update the branch in this `Gemfile` and merge it.

**Comment by jodosha** [comment]
:shipit:

**Pull request merged**